rust/src/dcerpc/detect.rs

```diff
@@ -19,7 +19,7 @@ use super::dcerpc::{
     DCERPCState, DCERPCTransaction, DCERPC_TYPE_REQUEST, DCERPC_TYPE_RESPONSE,
     DCERPC_UUID_ENTRY_FLAG_FF,
 };
-use crate::detect::{detect_match_uint, detect_parse_uint, DetectUintData};
+use crate::detect::detect::{detect_match_uint, detect_parse_uint, DetectUintData};
```

Comment:
I wonder if current detect.rs should be moved to detect/uint.rs

What do you think ?

Reply:
`detect.rs` contains logic for
- uint handling
- IP rep handling
- uri handling
- stream size

So, at a minimum, splitting `detect.rs` should be done into 4 modules.

I'll do this if you concur -- thoughts?